LongWarOfTheChosen/Config/XComLW_Overhaul.ini

```diff
@@ -510,6 +510,7 @@ SERIAL_AIM_MALUS_PER_KILL=0
 SERIAL_DAMAGE_FALLOFF=TRUE
 COVERING_FIRE_TO_HIT_BONUS_AGAINST_COVER=10
 FUSION_SWORD_FIRE_CHANCE=10
+BLUESCREEN_DISORIENT_CHANCE=50
```

Comment:
I'm not sold on the bluescreen changes. It feels like a balance change that isn't warranted pre 1.0. I'm not averse to the changes per se, but I refer back to wanting to be conservative as we approach a 1.0 release. And the psi changes are already big.